enabler/src/de/schildbach/pte/AustraliaProvider.java

```diff
@@ -32,6 +32,7 @@
     public static final String NETWORK_QLD = "TransLink";
     public static final String NETWORK_SA = "Adelaide Metro";
     public static final String NETWORK_WA = "Transperth";
+    public static final String NETWORK_ACT = "ACTION";
```

Comment:
What's that for?

Reply:
The Australia Provider didn't originally include the Canberra network, which is already a part of the ```au``` dataset on Navitia

Reply:
But this constant isn't used by anything. Also, you are adding a new provider it seems rather than modifying an existing one.

Reply:
In this case, it might very well be. Did you try to use the Australia provider in Transportr and check if it returns canberra results?

Reply:
Ahhh, yep it's working. I just tried the 2.0.0rc3 and it seems to be working just fine from preliminary searches. Is there any way for me to check all routes in Canberra? 

I think this PR is useless and can be closed now. Thanks for your help! :)

Reply:
> Is there any way for me to check all routes in Canberra?

No, PTE (this library) has no API for that.

> I think this PR is useless and can be closed now. Thanks for your help! :)

You should be able to close it yourself.